yellowbrick/features/scatter.py

```diff
@@ -0,0 +1,278 @@
+from yellowbrick.utils import is_dataframe
+from yellowbrick.exceptions import YellowbrickValueError
+from yellowbrick.style.colors import resolve_colors, get_color_cycle
+
+##########################################################################
+# Quick Methods
+##########################################################################
+
+
+def scatterviz(X,
+               y=None,
+               ax=None,
+               features=None,
+               classes=None,
+               color=None,
+               colormap=None,
+               markers=None,
+               **kwargs):
+    """Displays a bivariate scatter plot.
+
+    This helper function is a quick wrapper to utilize the ScatterVisualizer
+    (Transformer) for one-off analysis.
+
+    Parameters
+    ----------
+
+    X : ndarray or DataFrame of shape n x m
+        A matrix of n instances with m features
+
+    y : ndarray or Series of length n
+        An array or series of target or class values
+
+    ax : matplotlib axes
+        The axes to plot the figure on.
+
+    features : list of strings
+        The names of two features or columns.
+        More than that will raise an error.
+
+    classes : list of strings
+        The names of the classes in the target
+
+    color : list or tuple of colors
+        Specify the colors for each individual class
+
+    colormap : string or matplotlib cmap
+        Sequential colormap for continuous target
+
+    :param markers: iterable of strings
+        Matplotlib style markers for points on the scatter plot points
+
+    Returns
+    -------
+    ax : matplotlib axes
+        Returns the axes that the parallel coordinates were drawn on.
+    """
+    # Instantiate the visualizer
+    visualizer = ScatterVisualizer(ax, features, classes, color, colormap,
+                                   markers, **kwargs)
+
+    # Fit and transform the visualizer (calls draw)
+    visualizer.fit(X, y, **kwargs)
+    visualizer.transform(X)
+
+    # Return the axes object on the visualizer
+    return visualizer.ax
+
+
+##########################################################################
+# Static ScatterVisualizer Visualizer
+##########################################################################
+
+
+class ScatterVisualizer(DataVisualizer):
+    """
+    ScatterVisualizer is a bivariate feature data visualization algorithm that
+    plots using the Cartesian coordinates of each point.
+    """
+
+    def __init__(self,
+                 ax=None,
+                 features=None,
+                 classes=None,
+                 color=None,
+                 colormap=None,
+                 markers=None,
+                 **kwargs):
+        """
+        Initialize the base scatter with many of the options required in order
+        to make the visualization work.
+
+        Parameters
+        ----------
+
+        :param ax: the axis to plot the figure on.
+
+        :param features: a list of feature names to use
```

Comment:
Ok, then we should probably specify that in the docstring. 

In the future, would it be ok if I (or someone else) also added optional `x` and `y` keyword arguments in addition to the `features` argument such that either `x,y` must be specified or `features` may be specified, but not both? It would only change the signature of the function, not any other functionality.